compiler/options.nim

```diff
@@ -244,6 +244,7 @@ type
     searchPaths*: seq[AbsoluteDir]
     lazyPaths*: seq[AbsoluteDir]
     outFile*: RelativeFile
+    outFileAlt*: AbsoluteFile ## eg html produced by `nim doc`
```

Comment:
Why do we need this?

Reply:
fixed; now re-using `outFile` instead and it works